lighthouse-plugin-ad-speed-insights/utils/resource-classification.js

```diff
@@ -109,6 +109,16 @@ function isStaticRequest(request) {
   return ['parser', 'preload'].includes(request.initiator.type);
 }
 
+/**
+ * @param {LH.Crdp.DOM.BoxModel} iframe
+ * @param {?boolean} excludeNonVisible
+ * @return {boolean}
+ */
+function isGoogleAdsIframe(iframe, excludeNonVisible = true) {
```

Comment:
To clarify, this only covers GPT iframes. Other tags' iframes (e.g. AdSense) won't match this.

Reply:
-> isGPTIFrame()